windows/src/desktop/kmshell/main/initprog.pas

```diff
@@ -614,7 +614,7 @@ procedure ShowKeyboardWelcome(PackageName: WideString);  // I2569
 begin
   n := kmcom.Packages.IndexOf(PackageName);
   if n >= 0 then
-    DoShowPackageWelcome(kmcom.Packages[n], True);
+    DoShowPackageWelcome(nil, kmcom.Packages[n], True);
```

Comment:
Should we pass `Screen.ActiveForm` instead of `nil`?

Reply:
In this code path, `Screen.ActiveForm` will always be `nil` anyway.